src/app/components/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -1,21 +1,36 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import PropTypes, { string } from 'prop-types';
 import { Link } from 'react-router';
 import { bindActionCreators } from 'redux';
 import * as DS from '@nypl/design-system-react-components';
 import * as searchActions from '../../actions/SearchActions';
 import { MAX_TITLE_LENGTH } from '../../constants/editioncard';
+import EditionCard from '../Card/EditionCard';
 
-export const getBreadcrumbLinks = (workDetail) => {
+const getLinkFromWork = (title, uuid) => {
+  const workTitle = title || 'Title Unknown';
+
+  const strippedSlashTitle = workTitle.indexOf('/') > 0
+    ? workTitle.substring(0, workTitle.indexOf('/')) : workTitle;
+  const breadcrumbTitle = strippedSlashTitle.length > MAX_TITLE_LENGTH
+    ? `${strippedSlashTitle.substring(0, MAX_TITLE_LENGTH)}...` : strippedSlashTitle;
```

Comment:
Wonder if it makes sense to break the title here on a word boundary as well

Reply:
the word boundary ticket hasn't been done yet, but I will make a note to address breadcrumbs in that ticket!